docs/source/operations/configure_prerequisites.md

```diff
@@ -76,7 +81,7 @@ Build the Corda node image from **platforms/r3-corda/images** by following [thes
 <a name = "vaultunseal"></a>
 ## Unseal Hashicorp Vault 
 
-Hashicorp Vault is one of the pre-requisites for the Blockchain Automation Framework. If not initialised and unsealed already, complete the following steps to unseal and access the Vault.
+Hashicorp Vault is one of the pre-requisites for the Blockchain Automation Framework. Vault can be deployed as a kubernetes pod or on a machine and should be accessible by the ansible host as well as the kubernetes cluster (proper inbound/outbound rules should be configured). If not initialised and unsealed already, complete the following steps to unseal and access the Vault.
```

Comment:
"The Vault service should be accessible ..."
No need to mention kubernetes pod or a machine.